modules/sshkey/main.tf

```diff
@@ -21,7 +21,7 @@ resource "equinix_metal_ssh_key" "ssh_pub_key" {
 }
 
 resource "local_file" "cluster_private_key_pem" {
-  content         = chomp(tls_private_key.ssh_key_pair.private_key_openssh)
+  content         = chomp(tls_private_key.ssh_key_pair.private_key_pem)
```

Comment:
On MacOS 14.5, the openssh format was considered invalid. This change made the difference between a successful SSH in the `local-exec` and failure.

Reply:
We did the same thing here: https://github.com/equinix-labs/terraform-equinix-metal-nutanix-cluster/blob/main/modules/ssh/main.tf#L19-L24